tensorflow_addons/losses/tests/kappa_loss_test.py

```diff
@@ -90,3 +90,14 @@ def test_config():
 def test_serialization():
     loss = WeightedKappaLoss(num_classes=3)
     tf.keras.losses.deserialize(tf.keras.losses.serialize(loss))
+
+
+def test_save_model(tmpdir):
```

Comment:
Where `tmpdir` is coming for?

Reply:
it's purest fixture

https://docs.pytest.org/en/stable/tmpdir.html